tests/SdkTest.php

```diff
@@ -63,4 +63,26 @@ public function testCreatesClientsWithAlias()
             ])
         );
     }
+
+    public function testClonesWithExtraArgs()
+    {
+        $sdk = new Sdk([
+            'version' => 'latest',
+            'region'  => 'us-east-1',
+        ]);
+
+        $this->assertSame(
+            'us-east-1',
+            $sdk->createDynamoDb()->getRegion()
+        );
+
+        $copy = $sdk->copy([
```

Comment:
Can you add in a test to check that the copy keeps config options from the original (for options that were not overwritten)?